.yamllint.yaml

```diff
@@ -0,0 +1,31 @@
+---
+extends: default
+
+rules:
+  # 180 chars should be enough, but don't fail if a line is longer.
+  line-length:
+    max: 180
+    level: warning
+  comments:
+    min-spaces-from-content: 1
+  brackets:
+    forbid: false
+    min-spaces-inside: 1
+    max-spaces-inside: 1
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 1
+  braces:
+    forbid: false
+    min-spaces-inside: 0
+    max-spaces-inside: 1
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 1
+  document-start:
+    present: true
+    level: error
+  document-end:
+    present: false
+    level: error
+
+ignore:
+  - deployments/analytics-datadog/data/datadog.yaml
```

Comment:
why is that

Reply:
This looks like copy paste of config file from other repository, so every time we would like to update it with new version, it would need to be re-stailed again. So I think better approach is just to ignore it.